.travis.yml

```diff
@@ -0,0 +1,61 @@
+sudo: required
+dist: trusty
+os: linux
+language: minimal
+cache:
+  directories:
+  - contrib/depends/built
+  - contrib/depends/sdk-sources
+  - $HOME/.ccache
+env:
+  global:
+    - MAKEJOBS=-j3
```

Comment:
That should not be overridden (or there should be a user override).

Reply:
This is a travis CI instruction set. Why is the override problematic? I only included it, because it is quite useful, but basically it does not much more than the current build bot.

Reply:
Because building monero gobbles up a lot of RAM and 3 jobs means at least 6 GB.

Reply:
Removed MAKEJOBS.